setup.py

```diff
@@ -0,0 +1,21 @@
+# beprof setup.py
+
+from distutils.core import setup
+
+setup(
+    name = 'beprof',
+    packages = ['beprof'],
+    version = '0.1.0',
+    description = 'Beam profile analysing tool',
+    author = ['Leszek Grzanka', 'Mateusz Krakowski'],
+    author_email = 'grzanka@agh.edu.pl',
+    url = 'https://github.com/grzanka/beprof',
+    keywords = ["beam", "profile"],
+    classifiers = [
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Operating System :: OS Independent",
+        "Topic :: Scientific/Engineering :: Information Analysis",
+    ],
+    install_requires = ['numpy', 'scipy']
```

Comment:
sure right now working on scipy.
By the way which minimum version is required (or I should put up-to-date version)